frontend/ios/fastlane/Fastfile

```diff
@@ -85,13 +85,15 @@ platform :ios do
     # DO NOT OVERWRITE THE BUNDLE_CONFIG ENV VARIABLE! It is used by ruby bundle.
 
     sh("fvm flutter pub run build_runner build --define df_build_config=name=#{build_config_name}")
+    env_prod = sh("echo -n environment=production | base64")
 
     build_app(
        workspace: "Runner.xcworkspace",
        scheme: flavor,
        output_directory: "#{ENV['HOME']}",
        output_name: "#{build_config_name}.ipa",
        export_method: "app-store",
+       xcargs: "DART_DEFINES='#{env_prod}'"
```

Comment:
Maybe also remove the single-quotes? (Just guessing)
```suggestion
       xcargs: "DART_DEFINES=#{env_prod}"
```

Reply:
I tested this before. Didn't work. Thx for the hint with browserstack

Reply:
@michael-markl i had to pass the base64 string directly since somehow the result of this env_prod is wrong (LW4gZW52aXJvbm1lbnQ9cHJvZHVjdGlvbgo=)
Just decoded it and realized that it takes this for input `-n environment=production`

Reply:
I think it would be better to just adjust the command to handle the newline chars correctly. I think `echo environment=production | base64 | tr -d '\n'` should work.